repository/c.json

```diff
@@ -2987,6 +2987,17 @@
 				}
 			]
 		},
+		{
+			"name": "CoNLL-U.tmLanguage",
+			"details": "https://github.com/stephsamson/CoNLL-U.tmLanguage",
+			"labels": ["conllu", "ud", "universaldependencies", "language syntax"],
```

Comment:
The labels are for finding related packages. There probably won't be more `conllu` packages. There also don't seem to be any `ud` or `universaldependencies`, and it wasn't clear to me what these were or how they were related.